### PR TITLE
Started a small tips&tricks page

### PR DESCRIPTION
**Pull request by umlaeute**
a place for docs that go beyond the usual `make`

### CODE REVIEW

**Comment by katjav** [comment]
Good initiative, @umlaeute do you want to bring this in the master branch now or when the info is a bit more substantial?

**Comment by umlaeute** [comment]
i don't know when it will have grown enough to be "substential".

i think we should just merge it and add as we go.

**Comment by katjav** [comment]
Ok go ahead and merge. Can you write the section on 'Keeping pd-lib-builder uptodate?

File: tips-tricks.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+## cross-compiling W32 binaries from linux
+
+I'm using the following to cross-compile W32 binaries on my Debian/64bit system,
+using `mingw-w64`.
+
+Assuming you have unzipped a W32 package for Pd into `${WINPDPATH}`, run:
+
+    make system=Windows pdbinpath="${WINPDPATH}/bin/" pdincludepath="${WINPDPATH}/src/" CC=i686-w64-mingw32-gcc
+
+(if the project uses C++, you might also need to sed `CXX=i686-w64-mingw32-g++`)
```
Comment (residuum) [inline tips-tricks.md]: Typo: set

Comment (umlaeute) [reply tips-tricks.md]: fix it with `sed -e 's| sed | set |g'`

**Comment by umlaeute** [comment]
rebased & pushed to master

File: tips-tricks.md (hunk 1 above)
Comment (residuum) [reply tips-tricks.md]: This is `awk`ward

**Pull request closed without merging**